app/src/db/migrations/20200529145912_008-settings-request-access.js

```diff
@@ -0,0 +1,28 @@
+const PREFIX = require('../../forms/minesoperatorscreening/constants').PREFIX;
+
+exports.up = function(knex) {
+  return Promise.resolve()
+    .then(() => {
+      const data = {
+        name: 'accessRequestedEmail',
+        enabled: true,
+        config: {
+          template: 'access-requested-email.html',
+          from: 'MCAD.RegionalOps@gov.bc.ca',
+          to: 'NR.CommonServiceShowcase@gov.bc.ca',
+          subject: 'Mines Operator Screening - Access Requested',
```

Comment:
Did we change this in the other email templates?
On my local at least the subjects still refer to Industrial Camps

Reply:
changed